controllers/overlaynetwork_controller.go

```diff
@@ -47,9 +54,25 @@ type OverlayNetworkReconciler struct {
 // For more details, check Reconcile and its Result here:
 // - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.13.0/pkg/reconcile
 func (r *OverlayNetworkReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	_ = log.FromContext(ctx)
+	logger := log.FromContext(ctx).WithValues("overlaynetwork", req.NamespacedName)
+
+	logger.Info("Reconciling OverlayNetwork")
```

Comment:
I don't think we should have a log message each time the reconcile loop is invoked, it will overload the load.

Reply:
Done